test/validators.js

```diff
@@ -4291,6 +4294,9 @@ describe('Validators', () => {
         'rgba(3,3,3%,.3)',
         'rgb(101%,101%,101%)',
         'rgba(3%,3%,101%,0.3)',
+        'r         g    b(   0,         251,       222     )',
+        'r         g    ba(   0,         251,       222     )',
+        rg ba(0, 251, 22, 0.5)',
```

Comment:
Sorry, my suggestion was missing a quotation mark
```suggestion
        'rg ba(0, 251, 22, 0.5)',
```